docs/conf.py

```diff
@@ -29,7 +29,7 @@
 # The short X.Y version
 version = "1.0"
 # The full version, including alpha/beta/rc tags
-release = "1.0.0"
+release = "1.0.0pre1"
 
```

Comment:
Huh, I actually should make the docs read this out of the package as part of the "single source of truth" for the versioning.  I didn't think about that at the time.